src/templates/bootstrap/builderWizard/form.ejs

```diff
@@ -3,20 +3,18 @@
     {{ctx.sidebar}}
   </div>
   <div class="col-xs-8 col-sm-9 col-md-10 formarea">
-    <div class="breadcrumb wizard-page-container">
```

Comment:
Can we also have these changes made to @formio/bootstrap master branch?  This will contain all template changes from now on for the 5.x branch of the renderer.